MultiQuestensions/include/Packets/PacketManager.hpp

```diff
@@ -26,8 +26,8 @@ namespace MultiQuestensions {
 				return;
 			}
 
-			Il2CppString* identifier = packetType->ToString()->Replace(il2cpp_utils::createcsstr((std::string)"::"), il2cpp_utils::createcsstr((std::string)"."));
-			getLogger().info(to_utf8(csstrtostr(identifier)));
+			Il2CppString* identifier = packetType->ToString()->Replace(il2cpp_utils::newcsstr((std::string)"::"), il2cpp_utils::newcsstr((std::string)"."));
```

Comment:
Avoid casting to std::string unnecessarily, also consider converting the string to a c++ string and replacing characters on that instead.